test/paymentmethods.collective.to.collective.test.js

```diff
@@ -0,0 +1,438 @@
+      // Add User 1 as Collective 1 Admin
+      await models.Member.create({
+        CreatedByUserId: user1.id,
+        MemberCollectiveId: user1.CollectiveId,
+        CollectiveId: collective1.id,
+        role: 'ADMIN'
+      });
+
+      // Create stripe connected account to host of collective 1
+      await store.stripeConnectedAccount(collective1.HostCollectiveId);
+      // Add credit card to Collective 1
+      await models.PaymentMethod.create({
+        name: '4242',
+        service: 'stripe',
+        type: 'creditcard',
+        token: 'tok_123456781234567812345678',
+        CollectiveId: collective1.HostCollectiveId,
+        monthlyLimitPerMember: 10000
+      });
+
+      // Create stripe connected account to host of collective 3
+      await store.stripeConnectedAccount(collective3.HostCollectiveId);
+      // Add credit card to Collective 3
+      await models.PaymentMethod.create({
+        name: '4343',
+        service: 'stripe',
+        type: 'creditcard',
+        token: 'tok_123456781234567812345678',
+        CollectiveId: collective3.HostCollectiveId,
+        monthlyLimitPerMember: 10000
+      });
+
+      //finding opencollective payment method for Collective 1
+      openCollectivePaymentMethod = await models.PaymentMethod.findOne({ where: {type: 'collective', CollectiveId: collective1.id}});
+
+      // Setting up order with amount less than the credit card monthly limit
+      const order = {
+        fromCollective: { id: collective1.id },
+        collective: { id: collective3.id },
+        paymentMethod: { uuid: openCollectivePaymentMethod.uuid },
+        totalAmount: 1000,
+        interval: 'month',
+      }
+      // Executing queries
+      const res = await utils.graphqlQuery(createOrderQuery, { order }, user1);
+
+      // Then there should be no errors
+      res.errors && console.error(res.errors);
+      expect(res.errors).to.not.exist;
+
+      // When the order is created
+      // Then the created transaction should match the requested data
+      const orderCreated = res.data.createOrder;
+      const orderCreatedCollective = orderCreated.collective;
+      const orderCreatedFromCollective = orderCreated.fromCollective;
+      const subscription = orderCreated.subscription;
+      expect(subscription.interval).to.equal('month');
+      expect(subscription.isActive).to.be.true;
+      expect(subscription.amount).to.equal(order.totalAmount);
+
+      const transaction = await models.Transaction.findOne({
+        where: {
+          CollectiveId: orderCreatedCollective.id,
+          FromCollectiveId: orderCreatedFromCollective.id,
+          amount: order.totalAmount,
+        }
+      });
+      // make sure the transaction has been recorded
+      expect(transaction.FromCollectiveId).to.equal(collective1.id);
+      expect(transaction.CollectiveId).to.equal(collective3.id);
+      expect(transaction.currency).to.equal(collective1.currency);
+    });
```

Comment:
This is only testing the first transaction of the subscription. 
Can you also add a test (maybe in the subscription cron job tests?) to make sure that orders with an active subscription using the opencollective payment method are working across hosts?